ovos_plugin_common_play/ocp/utils.py

```diff
@@ -1,9 +1,27 @@
+import mimetypes
 import shutil
 import tempfile
 from os import makedirs
 from os.path import basename, expanduser, isfile, join, dirname
 
 from ovos_plugin_common_play.ocp.status import TrackState, PlaybackType
+from ovos_plugin_manager.ocp import StreamHandler
+
+ocp_plugins = StreamHandler()
+
+
+def find_mime(uri):
+    """ Determine mime type. """
+    mime = mimetypes.guess_type(uri)
+    if mime:
+        return mime
+    else:
+        return None
+
+
+def available_extractors():
+    return ["/", "http"] + \
```

Comment:
I'd like to make this more explicit as `file://`, `http://`, `https://` if we're just looking at string prefixes; we can handle parsing `/` to `file://` pretty easily and this makes it easier to handle more paths like `smb://`, `nfs://`, etc.